.github/workflows/build-fedora-toolbox.yml

```diff
@@ -75,15 +80,15 @@ jobs:
           password: ${{ env.REGISTRY_PASSWORD }}
           extra-args: |
             --disable-content-trust
```

Comment:
If I had to guess, this is because it would require us to trust the container locally before pushing it to the registry. In Red Hat's own workflows, for `push-to-registry`, they disable content trust:

https://github.com/search?q=repo%3Aredhat-actions%2Fpush-to-registry%20--disable-content-trust&type=code

However, I do believe this is something that would be worth looking into. At the very least, we may be able to emulate this behavior by signing the container locally with cosign before a push, and then verifying it after

Reply:
I would like to remove the flag.  It works without it in my older workflow that uses these actions.  Have made a PR

Reply:
Sounds great!